packages/theme_tailor/lib/src/generator/theme_tailor_generator.dart

```diff
@@ -1,73 +1,78 @@
+import 'dart:collection';
+
 import 'package:analyzer/dart/analysis/results.dart';
 import 'package:analyzer/dart/ast/ast.dart';
 import 'package:analyzer/dart/element/element.dart';
+import 'package:analyzer/dart/element/visitor.dart';
 import 'package:build/build.dart';
-import 'package:collection/collection.dart';
 import 'package:source_gen/source_gen.dart';
 import 'package:theme_tailor_annotation/theme_tailor_annotation.dart';
 
-import '../../theme_tailor.dart';
-import '../util/message.dart';
-import '../util/string_format.dart';
+import '../model/theme_extension_config.dart';
+import '../template/theme_extension_class_templates.dart';
```

Comment:
Czemu importy relative?

Reply:
Team #relativeImports można zmienić jeśli będzie taka potrzeba. Aktualnie w obu paczkach są relatywne

Reply:
to ja dodam swoje 3 grosze. Jak jest opcja żeby się ich pozbyć, to jestem za tym żeby były absolute

Reply:
Okay, zmienione na absolutne (w annotacjach i example też)